bcda/web/router.go

```diff
@@ -23,14 +25,9 @@ func NewAPIRouter() http.Handler {
 	}
 	FileServer(r, "/api/v1/swagger", http.Dir(swagger_path))
 
-	// Don't even load up the route in production
 	if os.Getenv("DEPLOYMENT_TARGET") != "prod" {
```

Comment:
Should `api.bcda.cms.gov` redirect to `bcda.cms.gov` too?  Totally open question, I'm not sure what the correct answer is.

Reply:
Totally fine with leaving this as-is for now. Really, this change is intended to capture existing references to `sandbox.bcda.cms.gov/` and funnel users to the new home of the user guide.